nilearn/input_data/multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,219 @@
+                 smoothing_fwhm=None, standardize=False, detrend=False,
+                 low_pass=None, high_pass=None, t_r=None, dtype=None,
+                 resampling_target="data",
+                 memory=Memory(cachedir=None, verbose=0), memory_level=1,
+                 n_jobs=1, verbose=0, strategy="mean"):
+        self.labels_img = labels_img
+        self.background_label = background_label
+        self.mask_img = mask_img
+
+        # Parameters for _smooth_array
+        self.smoothing_fwhm = smoothing_fwhm
+
+        # Parameters for clean()
+        self.standardize = standardize
+        self.detrend = detrend
+        self.low_pass = low_pass
+        self.high_pass = high_pass
+        self.t_r = t_r
+        self.dtype = dtype
+
+        # Parameters for resampling
+        self.resampling_target = resampling_target
+
+        # Parameters for joblib
+        self.memory = memory
+        self.memory_level = memory_level
+        self.n_jobs = n_jobs
+        self.verbose = verbose
+
+        available_reduction_strategies = {'mean', 'median', 'sum',
+                                          'minimum', 'maximum',
+                                          'standard_deviation', 'variance'}
+
+        if strategy not in available_reduction_strategies:
+            raise ValueError(str.format(
+                "Invalid strategy '{}'. Valid strategies are {}.",
+                strategy,
+                available_reduction_strategies
+            ))
+
+        self.strategy = strategy
+
+        if resampling_target not in ("labels", "data", None):
+            raise ValueError("invalid value for 'resampling_target' "
+                             "parameter: " + str(resampling_target))
+
+    def transform_imgs(self, imgs_list, confounds=None, n_jobs=1):
+        """Extract signals from a list of 4D niimgs.
+
+        Parameters
+        ----------
+        imgs_list: list of 4D Niimg-like objects
+            See http://nilearn.github.io/manipulating_images/input_output.html
+            Images to process. Each element of the list is a 4D image.
+
+        confounds: CSV file or array-like, optional
+            This parameter is passed to signal.clean. Please see the related
+            documentation for details.
+            shape: list of (number of scans, number of confounds)
+
+        Returns
+        -------
+        region_signals: list of 2D numpy.ndarray
+            List of signals for each label per subject.
+            shape: list of (number of scans, number of labels)
+        """
+        # We handle the resampling of labels separately because the affine of
+        # the labels image should not impact the extraction of the signal.
+
+        if not hasattr(self, 'mask_img_'):
+            raise ValueError('It seems that %s has not been fitted. '
+                             'You must call fit() before calling transform().'
+                             % self.__class__.__name__)
```

Comment:
```python
('It seems that {} has not been fitted. '
'You must call fit() before calling transform().'.format(
                             self.__class__.__name__)
)
```

Reply:
And so on, everywhere.

Reply:
Again, please note that this formatting inconsistencies are also present in the NiftiLabelsMasker object.

Reply:
Understood. Changing existing code is a different matter, but for new code, using the `'text {}'.format(str_var)` is the way to go.